components/org.wso2.carbon.identity.api.server.organization.management/org.wso2.carbon.identity.api.server.organization.management.v1/src/main/java/org/wso2/carbon/identity/api/server/organization/management/v1/service/OrganizationManagementService.java

```diff
@@ -737,7 +737,7 @@ private OrganizationsResponse getOrganizationsResponse(Integer limit, String aft
             }
             if (!isLastPage) {
                 String encodedString = Base64.getEncoder().encodeToString(organizations.get(organizations.size() - 1)
-                        .getCreated().getBytes(StandardCharsets.UTF_8));
+                        .getCreated().toString().getBytes(StandardCharsets.UTF_8));
```

Comment:
Remove .toString()

Reply:
This is needed as the `created` attribute in the Organization class is an Instant object and needs to be converted to a string.